cli/cmd/delete.go

```diff
@@ -66,8 +66,8 @@ var _deleteCmd = &cobra.Command{
 				exit.Error(err)
 			}
 		} else {
-			// local only supports deploying 1 replica at a time so _flagDeleteForce will be ignored
-			deleteResponse, err = local.Delete(args[0], _flagDeleteKeepCache)
+			// local only supports deploying 1 replica at a time, so _flagDeleteForce is only useful when APIs were deployed with different CLI version
```

Comment:
`is only useful when APIs were deployed with different CLI version` -> `is only useful when attempting to delete an API that has been deployed with different CLI version`